packages/orb/commands/notify.yml

```
@@ -130,4 +135,5 @@ steps:
         SLACK_STR_SHA256: "<<parameters.sha256>>"
         SLACK_STR_BIN_VERSION: "v0.2.7"
         SLACK_STR_BIN_OVERRIDE_URL: "<<parameters.bin_override_url>>"
+        SLACK_ORB_TIME_FORMAT: "<<parameters.time_format>>"
```

Should this be `SLACK_STR` to conform with standards?

```suggestion
        SLACK_STR_TIME_FORMAT: "<<parameters.time_format>>"
```

---

I was going to bring this one up. I'm not sure. The one difference here, is this env var is picked up by the CLI automatically, rather than set "internally". So I think the standard ones we use "internally" seem a bit cryptic if this were ever to be used as a standalone CLI. Open to either.

---

That's fair. Maybe `SLACK_CLI` to differentiate variables used strictly by the CLI?

---

I do want to avoid direct slack branding. so maybe SLACK_ORB_CLI